timemachine/fe/free_energy.py

```diff
@@ -557,7 +557,7 @@ def run_sims_sequential(
     for lamb_idx, initial_state in enumerate(initial_states):
 
         # run simulation
-        cur_frames, cur_boxes = sample(initial_state, md_params)
+        cur_frames, cur_boxes = sample(initial_state, md_params, max_buffer_frames=100)
```

Comment:
Same setting of `max_buffer_frames` we use for `run_sims_bisection`; prevents OOM for large systems.